torch/_dynamo/compiled_autograd.py

```diff
@@ -401,11 +416,12 @@ def runtime_wrapper(compiled_fn, inputs, sizes, scalars, hooks):
                 for i in runtime_inputs_to_move:
                     inputs[i] = inputs[i].pin_memory().cuda(non_blocking=True)
 
-                with _disable():
+                with _disable(), make_compile_context(self.id):
                     return compiled_fn(inputs, sizes, scalars, hooks)
             finally:
                 in_compiled_autograd_region = False
 
+        self.compile_context.__exit__(None, None, None)
```

Comment:
and this one is needed at compiled autograd compile time, to attribute compile artifacts to compiled id !0, and later to log compile time